Update rulesets for v2.0

xyhuang opened the pull request:
- Added v2.0 ruleset
- Moved verification and packaging scripts to /scripts

xyhuang commented:
Addresses #207

emizan76 reviewed (commented):
Can you add verify_for_v2.0_training.sh and change README.md?

shangw-nvidia commented on mlperf_logging/compliance_checker/training_2.0.0/common.yaml:
```diff
@@ -0,0 +1,149 @@
+    NAME:  submission_division
+    REQ:   EXACTLY_ONE
+    CHECK: " v['value'] in ['closed', 'open'] "
+    POST:  " enqueue_config('training_1.1.0/{}_common.yaml'.format(v['value'])) "
```
`raining_2.0.0` ?

xyhuang commented:
> Can you add verify_for_v2.0_training.sh and change README.md?

added, btw i also moved all the verify scripts to the /scripts sub dir, pls let me know if there's any concern.

xyhuang commented:
one another note: I did not change the following 2 files for ssd as they were introduced in a previous commit, pls let me know if any change is needed in those files:
- `mlperf_logging/compliance_checker/training_2.0.0/closed_ssd.yaml`
- `mlperf_logging/rcp_checker/training_2.0.0/rcps_ssd.json`
cc: @emizan76 @shangw-nvidia

shangw-nvidia commented:
> on another note: I did not change the following 2 files for ssd as they were introduced in a previous commit (#209, #211), pls let me know if any change is needed in those files:
> 
> * `mlperf_logging/compliance_checker/training_2.0.0/closed_ssd.yaml`
> * `mlperf_logging/rcp_checker/training_2.0.0/rcps_ssd.json`
> 
> cc: @emizan76 @shangw-nvidia

Those two files look fine to me; I don't see any (obvious) issues. Once this PR (and a few others that @emizan76 submitted) gets merged in, we can test it on the v1.1 submission to make sure.
